src/DevtoolsPanel/modules/CurrentState/CurrentState.jsx

```diff
@@ -25,9 +25,22 @@ const CurrentState = () => {
         fetchStateListeners();
     }, [fetchState, fetchStateListeners]);
 
+    const sortCurrentStateAlphabetically = useCallback(() => {
```

Comment:
Please use useMemo here instead of useCallback

Reply:
Done